armory/docker/management.py

```diff
@@ -21,22 +21,24 @@ class ArmoryInstance(object):
     def __init__(
         self, image_name, runtime: str = "runc", envs: dict = None, ports: dict = None
     ):
-        host_paths = HostPaths()
-        docker_paths = DockerPaths()
+        host_paths = paths.host()
+        docker_paths = paths.docker()
         self.docker_client = docker.from_env(version="auto")
 
         container_args = {
             "runtime": runtime,
             "remove": True,
             "detach": True,
             "volumes": {
-                host_paths.cwd: {"bind": "/workspace", "mode": "rw"},
-                host_paths.armory_dir: {"bind": docker_paths.armory_dir, "mode": "rw"},
+                host_paths.cwd: {"bind": docker_paths.cwd, "mode": "rw"},
+                #                host_paths.armory_dir: {"bind": docker_paths.armory_dir, "mode": "rw"},
```

Comment:
Nit -- this commented out line can be removed.

Reply:
Done.